test/explain/algorithm/test_gnn_explainer.py

```diff
@@ -31,10 +32,17 @@ def forward(self, x, edge_index, batch=None, edge_label_index=None):
             assert edge_label_index is not None
             x = x[edge_label_index[0]] * x[edge_label_index[1]]
 
-        if self.model_config.return_type.value == 'probs':
-            x = x.softmax(dim=-1)
-        elif self.model_config.return_type.value == 'log_probs':
-            x = x.log_softmax(dim=-1)
+        if self.model_config.mode.value == 'binary_classification':
+            x = x[:, 0]
```

Comment:
```suggestion
```
Let's drop this logic here to test unequal shapes of `y` and `y_hat` in the `GNNExplainer` (see comment above about `view_as`).